docs/general/options.md

```diff
@@ -0,0 +1,48 @@
+# Options
+
+## Scriptable Options
+
+Scriptable options also accept a function which is called for each data and that takes the unique argument `context` representing contextual information (see [option context](options.md#option-context)).
+
+Example:
+
+```javascript
+color: function(context) {
+    return context.data < 0 ? 'red' :   // draw negative values in red
+        index%2 ? 'blue' : 'green';    // else, alternate values in blue and green
+}
+```
+
+> **Note:** scriptable options are only supported by a few bubble chart options.
+
+## Indexable Options
+
+Indexable options also accept an array in which each item corresponds to the element at the same index. Note that this method requires to provide as many items as data, so, in most cases, using a [function](#scriptable-options) is more appropriated if supported.
+
+Example:
+
+```javascript
+color: [
+    'red',    // color for data at index 0
+    'blue',   // color for data at index 1
+    'green',  // color for data at index 2
+    'black',  // color for data at index 3
+    //...
+]
+```
+
+## Option Context
+
+The option context is used to give contextual information when resolving options and currently only applies to [scriptable options](#scriptable-options).
+
+The context object contains the following properties:
+
+- `datasetIndex`: index of the current dataset
+- `datasetCount`: number of datasets
+- `dataset`: dataset at index `datasetIndex`
+- `dataIndex`: index of the current data
+- `dataCount`: number of data
+- `data`: value of the current data
+- `chart`: the associated chart
```

Comment:
I think `chart, datasetIndex, dataIndex and dataset` is the minimum. The only reason to include the other properties is to save users code if they want to get those values.

I'm also OK with the minimal set of properties to keep our code smaller.